scripts/contiv-vpp-bug-report.sh

```diff
@@ -14,11 +14,20 @@ master_kubectl() {
     ssh "$SSH_USER@$MASTER" "${SSH_OPTS[@]}" kubectl "$@" || true
 }
 
+VPP_CMD_ERROR_LOG="vpp-cmd-errors.log"
 get_vpp_data() {
     echo " - vppctl '$1'"
+    echo "- vppctl '$1':" >> "$VPP_CMD_ERROR_LOG"
     # We need to call out /usr/bin/vppctl because /usr/local/bin/vppctl is a wrapper script that doesn't work inside the
     # container.
-    master_kubectl exec "$POD_NAME" -n kube-system -c contiv-vswitch /usr/bin/vppctl "$1" > "$2.log"
+    master_kubectl exec "$POD_NAME" -n kube-system -c contiv-vswitch /usr/bin/vppctl "$1" > "$2.log" 2>>"$VPP_CMD_ERROR_LOG"
```

Comment:
Having a single log for all errors in this fashion means we won't be able to figure out what command generated the error.

Reply:
The name of the command is echoed into the log file just before the error message. the resulting error log looks something like this:
- vppctl 'sh int':
command terminated with exit code 141
- vppctl 'sh int addr':
- vppctl 'sh ip fib':
command terminated with exit code 141
- vppctl 'sh l2fib verbose':
- vppctl 'sh ip arp':
command terminated with exit code 141
- vppctl 'sh vxlan tunnel':
- vppctl 'sh nat44 interfaces':
command terminated with exit code 141
- vppctl 'sh nat44 static mappings':
- vppctl 'sh nat44 addresses':
- vppctl 'sh nat44 deterministic mappings':
command terminated with exit code 141
- vppctl 'sh nat44 deterministic sessions':
command terminated with exit code 141
- vppctl 'sh nat44 deterministic timeouts':
command terminated with exit code 141
- vppctl 'sh nat44 hash tables detail':
command terminated with exit code 141
- vppctl 'sh nat44 sessions detail':
command terminated with exit code 141
- vppctl 'sh acl-plugin acl':
- vppctl 'sh hardware-interfaces':
command terminated with exit code 141
- vppctl 'sh errors':
command terminated with exit code 141

IMO, it's better to have all errors in one file, as long as you can identify them.

Reply:
Nitpick: spacing after >> should be consistent with the rest of the file:
```
2>> "$VPP_CMD_ERROR_LOG"
```